modules/ROOT/pages/values-and-types/index.adoc

```diff
@@ -0,0 +1,16 @@
+:description: This section provides an overview of data types in Cypher.
+= Values and types
+
+Cypher supports a range of data values. 
+When writing Cypher queries, it is not necessary to declare a data type. 
```

Comment:
It also isn't possible, this felt a little like you were implying it can be done, just doesn't need to be done

Reply:
Good point - that phrasing is unnecessarily ambiguous.